### PR TITLE
two file has modified

### PR DESCRIPTION
**Pull request by soumyasrimani**
please review below file has changed

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -3,4 +3,6 @@ writeline 2
 writeline 3
 writeline 4
 writeline 5
-writeline 6
\ No newline at end of file
+writeline 6
```
Comment (soumyasrimani16) [inline test.txt:6]: where the next line?

File: 3rd_text.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-this is thrid text file
\ No newline at end of file
+this is thrid text file
+this is another line added
```
Comment (soumyasrimani16) [inline 3rd_text.txt]: please add anothe line also

File: 3rd_text.txt [hunk 3]
```diff
@@ -1 +1,3 @@
...
+anothe line added
```
Comment (soumyasrimani16) [inline 3rd_text.txt]: I already changed.please go ahead.

**Review by soumyasrimani16 — APPROVED** [review]

**Comment by soumyasrimani** [comment]
changed

**Review by soumyasrimani — COMMENTED** [review]
please check again

**Pull request merged**